Tarea 2.1

juandiegovqdev opened the pull request.

Commit:
Update main.c

Commit:
Tarea 2.1

Commit:
Merge branch 'tarea1' into tarea2.1

Commit:
Tarea 2.1

He arreglado el archivo main.c, los errores.

Commit:
Tarea 2.2

Código modificado (he ampliado el trablero). Errores arreglados.

Commit:
Archivos innecesarios borrados.

juandiegovqdev committed:
main.ipch borrado

juandiegovqdev committed:
mmap_address.bin borrado

profedotc commented:
No puedo hacer merge. Hay conflicto porque en la tarea 2.1 usas `TAMX` en lugar de `TAM_X`

Commit:
Tarea 2.1

Nombre de constante actualizado (Ahora, en vez de llamarse TAMX o TAMY, se llama TAM_X o TAM_Y).

juandiegovqdev committed:
Archivo main.o eliminado.

juandiegovqdev committed:
Archivo gol.o eliminado.

profedotc commented:
Sigo sin poder hacer merge aquí. ¿necesitas ayuda?

profedotc commented:
Te aconsejo que te clones mi repositorio, `git clone https://github.com/profedotc/juan_villalobos.git repo_profe`, después aplica tus cambios sobre ese repositorio, uno a uno, y haces push al tuyo. Así evitaras los conflictos y ordenarás la tarea.

juandiegovqdev committed:
Merge branch 'master' into tarea2.1
